HARK/ConsumptionSaving/tests/test_ConsPortfolioModel.py

```diff
@@ -39,37 +39,47 @@ def test_solution(self):
     def test_sim_one_period(self):
         self.pcct.T_sim = 30
         self.pcct.AgentCount = 10
-        self.pcct.track_vars += ["aNrm"]
+        self.pcct.track_vars += ["aNrm", "mNrm", "bNrm", "TranShk", "cNrm"]
         self.pcct.initialize_sim()
 
         self.assertFalse(np.any(self.pcct.shocks["Adjust"]))
 
-        self.assertAlmostEqual(self.pcct.state_now["pLvl"][0], 1.0)
+        # simulation test -- seed/generator specific
+        # self.assertAlmostEqual(self.pcct.state_now["pLvl"][0], 1.0)
 
-        self.assertAlmostEqual(self.pcct.state_now["aNrm"][0], 7.257027956)
+        # simulation test -- seed/generator specific
+        # self.assertAlmostEqual(self.pcct.state_now["aNrm"][0], 7.257027956)
 
-        self.assertAlmostEqual(self.pcct.Rfree[0], 1.03)
+        # simulation test -- seed/generator specific
+        #self.assertAlmostEqual(self.pcct.Rfree[0], 1.03)
 
-        self.assertAlmostEqual(self.pcct.state_now["PlvlAgg"], 1.0)
+        # simulation test -- seed/generator specific
+        #self.assertAlmostEqual(self.pcct.state_now["PlvlAgg"], 1.0)
 
         self.pcct.sim_one_period()
 
-        self.assertAlmostEqual(self.pcct.state_now["pLvl"][0], 0.858934461)
-
-        self.assertAlmostEqual(self.pcct.state_now["aNrm"][0], 8.02359093)
-
-        self.assertAlmostEqual(self.pcct.shocks["PermShk"][0], 0.85893446)
-
-        self.assertAlmostEqual(self.pcct.shocks["TranShk"][0], 1.0)
+        self.assertAlmostEqual(
+            self.pcct.state_now["mNrm"][0],
+            self.pcct.state_now["bNrm"][0] + self.pcct.shocks["TranShk"][0],
+        )
 
-        self.assertAlmostEqual(self.pcct.shocks["Risky"][0], 0.92405816)
+        self.assertAlmostEqual(
+            self.pcct.controls["cNrm"][0],
+            self.pcct.solution[0].cFuncAdj(self.pcct.state_now["mNrm"][0]),
+        )
 
-        self.assertTrue(np.any(self.pcct.shocks["Adjust"][0]))
+        self.assertAlmostEqual(
+            self.pcct.controls["Share"][0],
+            self.pcct.solution[0].ShareFuncAdj(self.pcct.state_now["mNrm"][0]),
+        )
 
-        self.assertAlmostEqual(self.pcct.state_now["mNrm"][0], 9.70233892039)
+        self.assertAlmostEqual(
+            self.pcct.state_now["aNrm"][0],
+            self.pcct.state_now["mNrm"][0] - self.pcct.controls["cNrm"][0],
+        )
 
-        self.assertAlmostEqual(self.pcct.controls["Share"][0], 0.8627164488246847)
-        self.assertAlmostEqual(self.pcct.controls["cNrm"][0], 1.67874799)
+        # a drawn shock ; may not be robust to RNG/disitrubition implementations
```

Comment:
I'd remove this one? It tests a specific draw

Reply:
ok